sci-astronomy/calcmysky/calcmysky-0.3.3.ebuild

```diff
@@ -14,9 +14,9 @@ SRC_URI="
 LICENSE="GPL-3"
 # subslot is soversion
 SLOT="0/15"
-KEYWORDS="amd64 ~ppc ppc64 ~riscv ~x86"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~riscv ~x86"
 
-IUSE="qt5 qt6"
+IUSE="+qt5 qt6"
```

Comment:
Is there a specific reason to default to 5 here? At this point most packages are actually moving to 6 by default.

Reply:
I set it like this because stellarium doesn't recommend qt6 yet, because of feature parity. https://github.com/Stellarium/stellarium/issues/3473 has more details about one of issues, and in every release they find and fix more and more bugs with the telescope support in the qt6 build.

Reply:
The problem is that many profiles already have `USE=qt6` enabled globally, so if we now enable qt5 by default locally here, then the end result is that on these profiles we build both variants by default. That is probably not what we want.

Personally, at this point in the qt5 to qt6 transition I'd rather have no default then have qt5 be the default.

Reply:
Done